### PR TITLE
kcov: 35 -> 36

### PR DESCRIPTION
**Pull request by Ekleog**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/tools/analysis/kcov/aarch64_nt_prstatus.patch [hunk 1]
```diff
@@ -1,12 +0,0 @@
-diff --git a/src/engines/ptrace.cc b/src/engines/ptrace.cc
-index 59b615f..e02cddf 100644
---- a/src/engines/ptrace.cc
-+++ b/src/engines/ptrace.cc
-@@ -21,6 +21,7 @@
- 
- #if defined(__aarch64__)
- #  include <sys/uio.h>
-+#  include <elf.h>
```
Comment (Mic92) [inline pkgs/development/tools/analysis/kcov/aarch64_nt_prstatus.patch]: Have this been reported upstream?

**Comment by Mic92** [comment]
Don't do `sudo kcov --pid=1 /tmp/kcov` btw :)

**Comment by Ekleog** [comment]
Thank you for the quick merge! :) Though now I'm curious, what happened to you with `sudo kcov --pid=1 /tmp/kcov`? 😅

**Comment by Ekleog** [comment]
Oh wow.

**Pull request merged**